CLIKit/CLIApplication.h

```diff
@@ -140,6 +140,32 @@
  */
 - (NSString*)generateUsageMessage;
 
+/**
+ * Writes the usage message returned by `generateUsageMessage` to stderr.
+ */
+- (void)writeUseageMessage;
+
+/**
+ * Writes the localized description from the given error to stderr.
+ *
+ * @param error The error whose localized description to write to standard error.
+ */
+- (void)writeError:(NSError *)error;
+
+/**
+ * Write a given string to stderr.
+ *
+ * @param string The string to write to standard error.
+ */
+- (void)writeStdErrorString:(NSString *)string;
```

Comment:
@levigroker Should be renamed to - (void)writeToStandardError: (NSString*)string;